src/lib/Scenes/Home/Components/Sales/Components/SaleList.tsx

```diff
@@ -19,7 +19,9 @@ export class SaleList extends Component<any> {
           }}
           data={this.props.item.data}
           numColumns={numColumns}
-          keyExtractor={item => item.__id}
+          // keyExtractor={item => item.__id}
+          // FIXME:
+          keyExtractor={item => Math.random().toString()}
```

Comment:
Keys should be stable, what was the reason you couldn’t use e.g. `__id`?

Reply:
Oh this is a fixme just to get results to show up -- all WIP, was gonna uncomment above once ready.

Reply:
Gotcha 👍